octobot/initializer.py

```diff
@@ -28,9 +32,53 @@ def __init__(self, octobot):
         # Logger
         self.logger = get_logger(self.__class__.__name__)
 
+        self.interface_list = []
+        self.notifier_list = []
+
     async def create(self):
         # initialize tools
         self.__init_metrics()
+        await self._create_interfaces()
```

Comment:
I would prefer to have this two create methods in an services dedicated initializer file like `services_initializer.py`. What do you think about it ?

Reply:
In would rather call it ```interface_initializer.py``` service is an internal/tech term we use but I think we should refer to it as interfaces that includes the web/bots interfaces but also connexions to non exchanges websites. To me services does not imply all this, it's too generic.

Agree of a specific file